frontend/src/components/navigation/Header.tsx

```diff
@@ -22,6 +22,9 @@ const Header = () => {
     errors.forEach(error => dispatch(clear(error.name)));
     setShow(false);
   };
+  const url = `${window.location.protocol}//${window.location.hostname}${
+    window.location.port ? ':' + window.location.port : null
+  }`;
```

Comment:
Please refactor the above. It is adding "null" to the URL if no port is provided